.travis.yml

```diff
@@ -23,7 +23,7 @@ env:
   - SCRIPT="mvn surefire:test@unit-tests -T 2C"
   - SCRIPT="mvn surefire:test@integration-tests"
   - SCRIPT="mvn clean site --projects site-book"
-  - SCRIPT="mvn test --projects metron-interface/metron-config,metron-interface/metron-alerts"
+  - SCRIPT="./metron-interface/scripts/run-all-ui-test.sh"
```

Comment:
Migrated to bash script from maven. manven-frontend-plugin thrown error when tried to unzip Cypress.